benches/many_tasks.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
+
+use criterion::{criterion_group, criterion_main, Criterion};
+
+use tracing::instrument;
+use tracing_active_tree::layer;
+use tracing_subscriber::prelude::*;
+
+#[instrument]
+async fn dummy(n: usize) {
+    for _ in 0..10 {
+        tokio::task::yield_now().await;
```

Comment:
Need a `frame!` ?

Reply:
We can but I think it isn't necessary. The main goal of this case is for testing many small tasks being spawned.🤔